etc/kayobe/kolla/globals.yml

```diff
@@ -55,10 +55,10 @@ mariadb_tag: 7.0.2.1-1
 memcached_tag: 7.0.2.1-1
 monasca_tag: 7.1.0.1
 monasca_api_tag: 7.0.2.3-1
-monasca_grafana_tag: 7.0.2.5-1
+monasca_grafana_tag: 7.0.2.6-2
 monasca_log_api_tag: 7.0.2.2-1
 monasca_notification_tag: 7.0.2.2-1
-monasca_thresh_tag: 7.0.2.6-1
+monasca_thresh_tag: 7.0.2.6-2
```

Comment:
Hasn't the fix for this already been applied? We are running `7.0.2.6-1` for thresholder on both prod and alt-1.

Reply:
Right you are again! Removed the offending line.